src/panel.css

```diff
@@ -0,0 +1,30 @@
+.wrapper {
+  width: 280px;
+  height: 170px;
+  box-shadow: 0 8px 40px rgba(#000000, .2);
```

Comment:
Does CSS support hex for RGBA colors, or is that just a Sass thing?
Should this be rewritten as `rgba(0, 0, 0, .2)`?